build-tests/rush-tester/src/start.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
```

Comment:
Would there be a way to roll this launcher into the `@microsoft/rush` project?  That project conceptually functions as a front-end for `rush-lib`, and it does not create circular dependency the way that `rush-lib` does.

Reply:
Probably. I was thinking about that. Let me explore that more.